src/Tokenizer.js

```diff
@@ -52,16 +69,20 @@ export function token (type) {
 	switch (type) {
 		// * /
 		case 42: case 47:
-			return 3
+			return 5
```

Comment:
could you document those returned numbers anywhere? it's quite hard to grok them without knowing what they mean and how exactly they are supposed to be used